.vscode/settings.json

```diff
@@ -1,12 +1,21 @@
 {
     "typescript.tsdk": "node_modules/typescript/lib",
     "editor.formatOnSave": true,
+    "editor.codeActionsOnSave": {
+        "source.fixAll": true
```

Comment:
Code action is not guaranteed to be safe. If you auto safe it, it might include wrong fixes.

Reply:
I think it is think too much.

Reply:
IIRC React dependencies array will give code fixes but it might be wrong.